docs/EVG.md

```diff
@@ -760,6 +755,18 @@ Before issuing an EV Certificate, the CA MUST ensure that all Subject organizati
 
 As a general rule, the CA is responsible for taking all verification steps reasonably necessary to satisfy each of the Verification Requirements set forth in the subsections below.  The Acceptable Methods of Verification set forth in each of Sections 11.2 through 11.14 (which usually include alternatives) are considered to be the minimum acceptable level of verification required of the CA.  In all cases, however, the CA is responsible for taking any additional verification steps that may be reasonably necessary under the circumstances to satisfy the applicable Verification Requirement.
 
+### 11.1.3.  Disclosure of Verification Sources
+
+Effective as of 1 October 2020, prior to the use of an Incorporating Agency or Registration Agency to fulfill these verification requirements, the CA MUST publicly disclose, through an appropriate and readily accessible online means, Agency Information about the Incorporating Agency or Registration Agency.
+
+This Agency Information SHALL include at least the following:
+* Sufficient information to unambiguously identify the Incorporating Agency or Registration Agency (such as a name, jurisdiction, and website); and,
+* The accepted values for the `subject:jurisdictionLocalityName` (OID: 1.3.6.1.4.1.311.60.2.1.1), `subject:jurisdictionStateOrProvinceName` (OID: 1.3.6.1.4.1.311.60.2.1.2), and `subject:jursidictionCountryName` (OID: 1.3.6.1.4.1.311.60.2.1.3) fields when a certificate is issued using information from that Incorporating Agency or Registration Agency, indicating the jurisidction(s) that the Agency is appropriate for; and,
+* The accepted or allowed form or syntax of the Registration Number used by the Incorporating Agency or Registration Agency, if known; and,
+* A revision history that includes a unique version number and date of publication for any additions, modifications, or removals from this list.
```

Comment:
A date with version number is fine as long as CAs aren't obligated to provide a line by line log of everything that changed from one version to the next. To clarify this, we should delete: "additions, modifications, or removals from this list" and replace with "changes"

Reply:
Right. CAs aren't required to provide line-by-line logs, but versions need to be uniquely identifiable and have a sense of when they were published.

I originally had "changes", but the concern is that "changes" is ambiguous. A CA may not view an addition or removal as a change; that is, "change" is read as "modification". That's why I explicitly tried to enumerate all three, to make sure that it was unambiguous.

Do you think 'and/or' instead of 'or' would address that? To allow you to batch additions, modifications, and removals into a single contiguous version, rather than requiring a new version for each change if you make multiple changes at the same time?

I'm trying to find the balance between making it OK for CAs to batch changes, and to only have to maintain a distinct identifier for any document (so that folks can refer back to it), while also not having to add too much work. At the same time, I think there's a lot of benefit in knowing what was changed and removed (e.g. as [GLEIF](https://www.gleif.org/content/2-about-lei/7-code-lists/1-gleif-registration-authorities-list/2019-12-05_overview-changes-ra-list-v1.0-to-v1.5.pdf) publishes)

Reply:
Then we should say that.  The published list of Registration Agencies / Incorporating Agencies must be versioned and dated each time the list is modified.

Reply:
@dougbeattie There's a problem with stating only that as you propose, because it prevents historic comparison - e.g. the CA continually modifies a single URL and just bumps the version and date, without any understanding about what changed.

I'm not sure I'm fully understanding the challenge between adding Foo and adding an entry that says "Added foo" versus not having to do that. They seem... trivial and easily playbooked?

Reply:
It is important that it be possible to generate the version log, even if the published document is automatically generated from an internal database of approved sources.  It should simply require a date and version number bump whenever a new version is published.  In particular, it would be useful to make sure it is allowable to bump the date and version number even in the absence of changes (e.g. daily publication).

Reply:
@timfromdigicert I’m not sure I’m parsing you’re comment correctly. I’m reading the first sentence as “it should be possible to compare versions” and the second as “only a date and a version number need to be updated”, which seems at odds with the first?

Broadly, my high-level goal here is to understand the nature and frequency of changes, while providing as light a touch requirement as possible. I’m assuming that some CAs may implement this by publishing only their “current” version - e.g. if they’re using Google docs or a single PDF url or countless other forms - and don’t think it’s reasonable to require crawler/scrapers be developed to constantly poll 24/7 to download versions so they can be diff’d.

If the CA uses a manual process, such as human review, having the human annotate the changes in a Changelog provides that data.

If the CA uses an automated process, such as something like Github or even direct CA integration, then the existing diffs themselves contain the data.

In the worst case, if using a PDF, the CA could accomplish this requirement by publishing their new version, running a PDF diff with the old, and just adding that PDF-diff to their changelog.

That’s the high-level goal, but if there are ways to accomplish that while addressing concerns for non-goals, I’m definitely open.

Reply:
I'm not too picky about the mechanics either, I just want to make sure a potential use case where the document is auto-generated, say, daily, is compliant.

Such a system might generate Changelog entries such as:

2020-05-15 - Version 2020.05.15 - No changes.

I just want to make sure that the language does not preclude such "updates", even if they do not include any additions, removals, or modifications.  I would argue that's already allowed by your current language, but wanted to explicitly point it out in case anyone disagrees.

Reply:
```suggestion
* A revision history that includes a unique version number and date of publication for any additions, modifications, and/or removals from this list.
```

This at least attempts to address part of @dougbeattie 's concern, by making these updates batchable upon publication.